SimpleCam/SimpleCam/SimpleCam.m

```diff
@@ -526,7 +560,8 @@ - (void) capturePhoto {
          }
          
          isCapturingImage = NO;
-         _capturedImageV.image = capturedImage;
+         _capturedImageV.alpha = 0.0f;
```

Comment:
I don't see the counter where this is set back to 1, did I miss it somewhere?  I don't have a device on hand to build this, so I'm reviewing blind at the moment.